server/src/test/java/com/broadcom/lsp/cobol/usecases/engine/UseCaseEngine.java

```diff
@@ -209,10 +210,14 @@ private static void assertResultEquals(AnalysisResult actual, TestData expected)
     assertResult("Constant usages:", expected.getConstantUsages(), actual.getConstantUsages());
 
     assertResult(
-        "Paragraph definition:",
-        expected.getParagraphDefinitions(),
-        actual.getParagraphDefinitions());
-    assertResult("Paragraph usage:", expected.getParagraphUsages(), actual.getParagraphUsages());
+        "Paragraph definition:", expected.getParagraphDefinitions(), actual.getParagraphDefinitions());
+    assertResult("Paragraph usages:", expected.getParagraphUsages(), actual.getParagraphUsages());
+
+    if (!expected.getSectionDefinitions().isEmpty()) {
```

Comment:
You don't need this condition, the assertion should be applied in any case

Reply:
Thanks for help! Closed.